### DISCUSSION TITLE
Code server redirection

### DISCUSSION THREAD

**Discussion by mohammedyusha** [coder/coder#4475] [General] [answered]
Is it possible for the users to be redirected directly to the code server in place of showing Dashboard (workspace, templates, users). Basically, the users should not see the Coder Web UI instead they should be redirected to the code server after signing in.

**Comment by jsjoeio** [comment, marked as answer]
So I tried a couple things and it looks like it is possible:

If you're signed out and try to navigate directly to code-server i.e. (https://dev.coder.com/@jsjoeio/joe-coder.dev/apps/code-server/), it will show you the login page. After login, it will auto-redirect to Coder Web UI.

Here's a video demo:

https://user-images.githubusercontent.com/3806031/195653317-81f0efc4-bcf7-48a5-9b5a-abaa627cc0cd.mov

Bookmark the URL for Coder Web UI, and everything else should just work!
